sw/device/silicon_creator/rom_ext/sival/binaries/BUILD

```diff
@@ -46,3 +47,16 @@ exec_env_filegroup(
         "rom_ext_dice_cwt_prod_slot_virtual_silicon_creator.signed.bin": "silicon",
     },
 )
+
+filegroup(
+    name = "signed_binaries",
+    srcs = glob(["*.signed.bin"]),
+    visibility = ["//visibility:public"], # or specific packages
```

Comment:
You don't need `visibility` here because this BUILD file already declares visibility for this package.